README.md

```diff
@@ -4,3 +4,21 @@ DUB registry
 ![vibe.d logo](public/images/logo-small.png) Online registry for [dub](https://github.com/dlang/dub/) packages, see <http://code.dlang.org/>.
 
 [![Build Status](https://travis-ci.org/dlang/dub-registry.svg)](https://travis-ci.org/dlang/dub-registry)
+
+Deploy your own mirror
+----------------------
+
+[![Deploy your own Mirror](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/dlang/dub-registry)
+
+After you have added your mirror, you should configure a daily or hourly cron job.
+Select the "Scheduler" addon and add the following task:
+
+```
+./dub-registry --mirror=https://code.dlang.org --run-cron --vv
```

Comment:
I'd say the private registry use-case is way more interesting than encouraging dozens of mirrors, in particular with the current mirror protocol.